hooks/pre_gen_project.py

```diff
@@ -1,13 +1,16 @@
 import re
 import sys
 
+MODULE_REGEX = r"^[_a-zA-Z][_a-zA-Z0-9]+$"
 
-MODULE_REGEX = r'^[_a-zA-Z][_a-zA-Z0-9]+$'
-
-module_name = '{{ cookiecutter.package_slug}}'
+module_name = "{{ cookiecutter.package_slug}}"
 
 if not re.match(MODULE_REGEX, module_name):
-    print('ERROR: The project slug (%s) is not a valid Python module name. Please do not use a - and use _ instead' % module_name)
+    print(
+        "ERROR: The project slug (%s) is not a valid    \
```

Comment:
maybe something like this would be better:

```python
    print(
        "ERROR: The project slug (%s) is not a valid Python module name."
        "Please do not use a - and use _ instead" % module_name
    )